tox.ini

```diff
@@ -51,6 +54,7 @@ deps =
     dj111: Django<1.12
```

Comment:
There are still some references to Django 1.11 in `tox.ini`. This is a good time to remove those too.